mappings/net/minecraft/client/gl/GlUniform.mapping

```diff
@@ -0,0 +1,34 @@
+CLASS none/bwp net/minecraft/client/gl/GlUniform
+	FIELD a LOGGER Lorg/apache/logging/log4j/Logger;
+	FIELD b loc I
+	FIELD c count I
+	FIELD d dataType I
+	FIELD e intData Ljava/nio/IntBuffer;
+	FIELD f floatData Ljava/nio/FloatBuffer;
+	FIELD g name Ljava/lang/String;
+	FIELD h stateIsDirty Z
+	FIELD i program Lnone/bwk;
+	METHOD <init> (Ljava/lang/String;IILnone/bwk;)V
+		ARG 0 name
+		ARG 1 dataType
+		ARG 2 count
+		ARG 3 program
+	METHOD a getName ()Ljava/lang/String;
+	METHOD a setFloat1 (F)V
```

Comment:
This is an abstraction layer over direct OpenGL calls, and since we have overloads, we should make use of it, in my opinion. (It's what OpenTK does, too, for example.)